plugins/MauticCrmBundle/Api/SalesforceApi.php

```diff
@@ -337,17 +346,7 @@ public function getLeads($query, $object)
             $fields[] = 'Id';
             $fields   = implode(', ', array_unique($fields));
```

Comment:
$fields are passed to to requestQueryAllAndHandle method and should be array

This return error

`Type error: Argument 2 passed to MauticPlugin\MauticCrmBundle\Api\SalesforceApi::requestQueryAllAndHandle() must be of the type array, string given`